src/test/java/com/develhope/spring/deals/services/OrderServiceTest.java

```diff
@@ -43,48 +34,27 @@ public void setup() {
     @MockBean
     private OrderRepository orderRepository;
 
+    @MockBean
+    private OrderMapper orderMapper;
+
     @Autowired
     private OrderService orderService;
 
+    @Test
+    void deleteOrder_ShouldDeleteOrder_WhenOrderExists() {
+        Long orderId = 1L;
 

```

Comment:
dove in questo test fai il setup dell'ambiente per il test? Per testare che delete funzioni, dovresti prima setuppare una repository dove viene salvato almeno un order